inference-engine/src/multi_device/multi_device_exec_network.hpp

```diff
@@ -37,6 +43,12 @@ struct DeviceInformation {
     std::map<std::string, std::string> config;
     int numRequestsPerDevices;
     std::string defaultDeviceID;
+    DeviceName uniqueName;
```

Comment:
const, I guess

Reply:
I try to add const for uniqueName , but meet compile issue, the issue happen when compile DeviceInformation a = b,
for example _loadContext[ACTUALDEVICE].deviceInfo =  _loadContext[CPU].deviceInfo